processes/pagerduty_process.md

```diff
@@ -0,0 +1,16 @@
+# PagerDuty Workflow
+
+PagerDuty is Hack Club's on-call system and issue escalation policy. It works by scheduling turns for engineers to be responsible for resolving any problem that occurs in production.
+
+## Create An Account
+
+The first step is to create a PagerDuty account. You can read the full instructions [here][pagerduty_create_account].
+Make sure to install the app for your phone and have notifications all set up before going to the next step.
+
+Join the team by asking a staff member with a PagerDuty account to invite you to be in the PagerDuty team.
+
+[pagerduty_create_account]: https://support.pagerduty.com/hc/en-us/articles/202828690-PagerDuty-Quick-Start-Guide#userprofile
+
+## On-Call
```

Comment:
It's certainly connected, but it's out of the scope of our PagerDuty process. Just like how it would be out of scope to document [how to create a PR](https://help.github.com/articles/about-pull-requests/) in the PagerDuty process doc, even though it's related.